tests/boardfarm_plugins/boardfarm_prplmesh/prplmesh_config.json

```diff
@@ -4,6 +4,17 @@
         "board_type": "prplmesh_docker",
         "role": "controller",
         "conn_cmd": "",
-        "devices": []
+        "devices": [
+            {
+                "name": "repeater1",
+                "type": "prplmesh_docker",
```

Comment:
Actually, I believe the idea was to have repeater1 as DUT, and the controller as a peripheral device. So it would make more sense to structure it as
```
{
    "prplmesh_docker": {
        "name": "repeater1",
        "role": "agent",
        ...
       "devices": [
          {
            "name": "gateway",
            "role": "controller",
...
```

But this doesn't make any practical difference, so (nitpick).